src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -420,7 +420,7 @@ public static class StandardTrainersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with <see cref="AveragedPerceptronTrainer"/> and advanced options.
+        /// Create an <see cref="AveragedPerceptronTrainer"/> with advanced options, which predicts a target using a linear binary classification model trained over boolean label data.
```

Comment:
>over boolean label data [](start = 160, length = 23)

over a label of boolean data.

Reply:
Natalie voted for no change.

---
In reply to: [275001284](https://github.com/dotnet/machinelearning/pull/3310#discussion_r275001284) [](ancestors = 275001284)